### DISCUSSION TITLE
Having trouble in running download_contents.py

### DISCUSSION THREAD

**Discussion by sundayMH** [AI4Finance-Foundation/FinGPT#49] [Q&A] [answered]
I can run download_titles.py correctly. And save these files in my file folder.
![image](https://github.com/AI4Finance-Foundation/FinGPT/assets/139118706/21ab97c8-be15-4f0b-beb5-c1b8e176c968)

But when I run download_contents.py, the return in Pycharm like this
![image](https://github.com/AI4Finance-Foundation/FinGPT/assets/139118706/8737e3fc-8738-40b9-881d-b99a94c75b53)
and I can't find the files in my file folder.

I need your help!!

**Comment by oliverwang15** [comment, marked as answer]
This may be the same reason as [this issue](https://github.com/AI4Finance-Foundation/FinGPT/issues/43#issue-1782195658) which is a problem of the anti-scraping policy of the source website. So you may need ip pools to fetch the content. BTW, I have updated the [demo notebook](https://github.com/AI4Finance-Foundation/FinGPT/blob/master/fingpt/FinGPT-v1/data_preparations/download_content_demo.ipynb) to show that the function can fetch the data successfully.

**Reply by sundayMH** [reply]
Thanks for your answer, I also use Kuaidaili to change my ip, and I will try it again. 
如果不嫌麻烦的话，也可以将数据集压缩上传，或者我也可以尝试使用自己的数据
